src/main/java/net/dv8tion/jda/internal/interactions/command/CommandImpl.java

```diff
@@ -221,6 +221,14 @@ public long getIdLong()
         return id;
     }
 
+    @Nonnull
+    @Override
+    public String getAsMention()
+    {
+        Checks.check(getType() == Type.SLASH, "Only slash commands can be mentioned");
```

Comment:
This shouldn't use `Checks.check` and instead use `throw new IllegalStateException`. Checks is only for arguments.